src/Symfony/Component/PropertyAccess/PropertyAccessorInterface.php

```diff
@@ -111,4 +111,11 @@ public function isWritable($objectOrArray, $propertyPath);
      * @throws Exception\InvalidArgumentException If the property path is invalid
      */
     public function isReadable($objectOrArray, $propertyPath);
+
+    /**
+     * Returns the list of values from an array of data and/or objects.
+     *
+     * @param string|PropertyPathInterface $propertyPath
+     */
+    public function getValues(array $array, $propertyPath): array;
```

Comment:
Adding a new method on the interface directly is not backward compatible. I guess the first step would be to add it as a "virtual" `@method` on 4.4 and document it in CHANGELOG and UPGRADE.